docs: update TTFB graph explainer (#85)
plprobelab/website: content.en/websites/_index.md

performance differences and provides insights into the efficiency of each
protocol/implementation in different geographic areas.

For Helia in the above graph, the different variations work out as follows:

> `Helia (TG)` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) only

Reviewer: ```suggestion
`Helia (TG)` - Helia in NodeJS delegating [content routing](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs), [peer routing](https://docs.ipfs.tech/concepts/glossary/#peer-routing), and block retrieval entirely to [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/).  js-libp2p is not used.  This is the simplest case scenario for the Helia node since an external server is doing all of the heavy lifting.
```

> `Helia (DR)` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) only
> `Helia (TG)` - Helia in NodeJS: blocks fetched via [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval only 

Author: ```suggestion
`Helia (TG)` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) only
`Helia (DR)` - Helia in NodeJS: blocks fetched via [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval only 
```

> `Helia (DR)` - Helia in NodeJS: blocks fetched via [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval only 

Reviewer: ```suggestion
`Helia (DR)` - Helia in NodeJS that [delegates content and peer routing to an external server using HTTP](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs) but then uses js-libp2p for direct peer retrieval.  (HTTP Trustless Gateways are not used.)
```

> `Helia (TG)` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) only
> `Helia (DR)` - Helia in NodeJS: blocks fetched via [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval only 
> `Helia` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) & [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval

Author: Fix nbsp issue: 
![image](https://github.com/plprobelab/website/assets/1173416/1c58d044-0c23-4791-a76d-7c746a152823)

```suggestion
- `Helia (TG)` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) only
- `Helia (DR)` - Helia in NodeJS: blocks fetched via [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval only 
- `Helia` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) & [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval
```

> `Helia` - Helia in NodeJS: blocks fetched via [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) & [delegated routing](https://specs.ipfs.tech/routing/http-routing-v1/) with direct peer retrieval

Reviewer: ```suggestion
`Helia` - Effectively the combination of "Helia (TG)" + "Helia (DR)".  Helia in NodeJS in parallel is:
 - delegating content, peer routing, and block entirely to [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) AND
 - delegating content and peer routing to an external server using HTTP but then uses js-libp2p for direct peer retrieval. 
It's expected that this would yield more reliable results than "Helia (TG)" or "Helia (DR)" alone.

Important scenarios that aren't measured yet:
1. Helia in NodeJS where content, peer, and block retrieval are done entirely in a p2p fashion.  
2. The scenarios above but instead from within Chromium rather than NodeJS.  This will be coming soon per ISSUE X.
```

performance differences and provides insights into the efficiency of each
protocol/implementation in different geographic areas.

For Helia in the above graph, the different variations work out as follows:

> `Helia (TG)` - Helia in NodeJS delegating [content routing](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs), [peer routing](https://docs.ipfs.tech/concepts/glossary/#peer-routing), and block retrieval entirely to [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/).  js-libp2p is not used.  This is the simplest case scenario for the Helia node since an external server is doing all of the heavy lifting.
> `Helia (DR)` - Helia in NodeJS that [delegates content and peer routing to an external server using HTTP](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs) but then uses js-libp2p for direct peer retrieval.  (HTTP Trustless Gateways are not used.)
> `Helia` - Effectively the combination of "Helia (TG)" + "Helia (DR)".  Helia in NodeJS in parallel is:

Author: ```suggestion

`Helia (TG)` - Helia in NodeJS delegating [content routing](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs), [peer routing](https://docs.ipfs.tech/concepts/glossary/#peer-routing), and block retrieval entirely to [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/).  js-libp2p is not used.  This is the simplest case scenario for the Helia node since an external server is doing all of the heavy lifting.

`Helia (DR)` - Helia in NodeJS that [delegates content and peer routing to an external server using HTTP](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs) but then uses js-libp2p for direct peer retrieval.  (HTTP Trustless Gateways are not used.)

`Helia` - Effectively the combination of "Helia (TG)" + "Helia (DR)".  Helia in NodeJS in parallel is:
```

protocol/implementation in different geographic areas.

For Helia in the above graph, the different variations work out as follows:

`Helia (TG)` - Helia in NodeJS delegating [content routing](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs), [peer routing](https://docs.ipfs.tech/concepts/glossary/#peer-routing), and block retrieval entirely to [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/).  js-libp2p is not used.  This is the simplest case scenario for the Helia node since an external server is doing all of the heavy lifting.

`Helia (DR)` - Helia in NodeJS that [delegates content and peer routing to an external server using HTTP](https://docs.ipfs.tech/concepts/how-ipfs-works/#how-content-routing-works-in-ipfs) but then uses js-libp2p for direct peer retrieval.  (HTTP Trustless Gateways are not used.)

`Helia` - Effectively the combination of "Helia (TG)" + "Helia (DR)".  Helia in NodeJS in parallel is:
 - delegating content, peer routing, and block entirely to [trustless gateways](https://specs.ipfs.tech/http-gateways/trustless-gateway/) AND

>  - delegating content and peer routing to an external server using HTTP but then uses js-libp2p for direct peer retrieval. 

Author: block retrieval. good eye

>  - delegating content and peer routing to an external server using HTTP but then uses js-libp2p for direct peer retrieval. 

Author: well, it's direct peer retrieval of a block..
